### PR TITLE
add fillet rect

### PR DESCRIPTION
**Pull request by trp02**
Related to issue #5741 
Not ready to merge just needed some help. 

I mostly followed the same method as #5817 and the other shapes. However, when it tries to draw the [fillet rectangle](https://api.pixijs.io/@pixi/graphics-extras/src/drawFilletRect.ts.html) it gives me an error and I can't seem to figure out why. Any help is appreciated. 

image

image

### CODE REVIEW

File: Extensions/PrimitiveDrawing/Extension.cpp [hunk 1]
```diff
@@ -124,6 +124,25 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
...
 
+  obj.AddAction("FilletRectangle",
+              _("Fillet Rectangle"),
+              _("Draw a fillet rectangle on screen"),
+              _("Draw from _PARAM1_;_PARAM2_ to _PARAM3_;_PARAM4_ a fillet "
+                "rectangle (fillet: _PARAM5_)" 
+                "with _PARAM0_"),
+                _("Drawing"),
+                "res/actions/roundedRectangle24.png",
+                "res/actions/roundedRectangle.png")
```
Comment (trp02) [inline Extensions/PrimitiveDrawing/Extension.cpp:135]: Just placeholders. Will create custom pngs once everything else is implemented.

**Comment by AlexandreSi** [comment]
It's a weird error. Did you build GDevelop.js and are you developing with Electron running (so that it builds GDJS as well)?

**Comment by trp02** [comment]
I built GDevelop.js but wasn't running on Electron. I'll try that.

**Comment by trp02** [comment]
That worked, thank you! Here are images of a rounded_rectangle as well as a fillet_rectangle with the same parameters and it seems to be the same for positive fillet values. 
image
image

I was thinking about it from a user friendliness perspective and even though drawFillet can do the job of drawRounded, people may not know what a fillet rectangle is(I know I didn't) so keeping both options may be most accessible. Interested to hear your thoughts on this.

File: Extensions/PrimitiveDrawing/Extension.cpp [hunk 2]
```diff
@@ -140,7 +159,7 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("expression", _("Right X position"))
       .AddParameter("expression", _("Bottom Y position"))
       .AddParameter("expression", _("Radius (in pixels)"))
-      .SetFunctionName("DrawRoundedRectangle");
+      .SetFunctionName("DrawRoundedRectangle");
```
Comment (AlexandreSi) [inline Extensions/PrimitiveDrawing/Extension.cpp:162]:
```suggestion
      .SetFunctionName("DrawRoundedRectangle");
```

File: Extensions/PrimitiveDrawing/Extension.cpp [hunk 3]
```diff
@@ -124,6 +124,25 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
...
+                "res/actions/filletRectangle24.png",
+                "res/actions/filletRectangle.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("expression", _("Left X position"))
+      .AddParameter("expression", _("Top Y position"))
+      .AddParameter("expression", _("Right X position"))
+      .AddParameter("expression", _("Bottom Y position"))
+      .AddParameter("expression", _("Fillet (in pixels)"))
+      .SetFunctionName("DrawFilletRectangle");
```
Comment (AlexandreSi) [inline Extensions/PrimitiveDrawing/Extension.cpp:143]:
```suggestion
      .SetFunctionName("DrawFilletRectangle");
```

**Review by AlexandreSi — APPROVED** [review]
LGTM!
Thanks for implementing all those shapes!

**Pull request merged**